### PR TITLE
Space bar plays/pauses music in queue view

### LINKED ISSUE #4857: Inconsistent space key behavior
**Describe The Bug**
The behavior when hitting the space bar is inconsistent between video and music playback

**Steps To Reproduce**
1. Start playing music
2. Press space key

**Expected Behavior**
Music should pause, then start playing again upon the space key being pressed again.

**Actual Behavior**
After starting playback, the space key will do one of 2 things:
1. If I had clicked anything prior to pressing the space key, hitting the space key will act as if said button has been clicked again.
examples:
   - If I hit space right after starting an album, the space key will act as if I clicked play on the album again and start the album over from the start
   - If I click on next track then hit space, it will skip to the next track.

2. If I had not clicked anything on that page, hitting the space key will scroll down.

**System (please complete the following information):**
 - Platform: MacOS
 - Browser: JMP, Chrome, Firefox
 - Jellyfin Version: 10.8.11

**Additional Context**
During video playback, hitting the space key always pauses / unpauses playback regardless of what I had clicked prior to pressing the space key.

### PR DESCRIPTION
**Pull request by GAntoine**
**Changes**

This PR overrides the behavior of the space bar (when in queue view) to play/pause the player. The behavior and code were essentially lifted from `playback/video` but stripped down to the bare essentials.

**Issues**

Closes #4857

### CODE REVIEW

**Comment by thornbill** [comment]
Hey @GAntoine, just for future reference, we don't typically include issue numbers in commit messages / PR titles. We strive to follow [these guidelines](https://cbea.ms/git-commit/). Additional details can be found in [our docs](https://jellyfin.org/docs/general/contributing/development#pull-request-guidelines). Thanks!

File: src/controllers/playback/queue/index.js [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
+    function releaseCurrentPlayer() {
+        const player = currentPlayer;
+        if (player) currentPlayer = null;
+    }
+
+    function bindToPlayer(player) {
+        if (player !== currentPlayer) {
+            releaseCurrentPlayer();
+            currentPlayer = player;
+            if (!player) return;
```
Comment (thornbill) [inline src/controllers/playback/queue/index.js:29]: This seems unnecessary?

**Comment by GAntoine** [comment]
Hey thanks for the heads up, I've gone ahead and updated the title. I use it as a way to link the linear/jira/trello ticket to a PR, and I guess it's become a habit lol. I'll give those link a quick read!

File: src/controllers/playback/queue/index.js (hunk 1 above)
Comment (GAntoine) [reply src/controllers/playback/queue/index.js:29]: Yeah it can be removed, I copied a lot of this and it must have slipped through.

**Review by thornbill — APPROVED** [review]
:+1: appears to work as expected!

**Pull request merged**